generator/testdata/expected_validation.go

```diff
@@ -9,13 +9,15 @@ import (
 	"runtime/debug"
 	"sync"
 
+	githubproto "github.com/golang/protobuf/proto"
```

Comment:
Is there a reason you created alias ? I would like to use the original packge name as much as possible.

Reply:
That actually conflicts with ["google.golang.org/protobuf/proto"](https://github.com/mercari/grpc-federation/blob/7ec704f932af243ba27a311bac37c1d47a6900f7/generator/templates/server.go.tmpl#L17)...

Reply:
naruhodo. ( Message V1 and V2 problem...)
Oh, `google.golang.org/protobuf/proto` does not seem to be used, so it may be deleted.